demo/src/ExamplesLoader.js

```diff
@@ -13,8 +13,18 @@ export const loadExamples = () => {
     <div class="category" data-category="${category}">
         <h2>${category}</h2>
         ${examples.map(example => `
-          ${example.description != null ? `<p>${example.description}</p>` : ''}
-          <button class="btn example-btn">${example.title}</button>
+          <button class="btn example-btn">
+            ${
+              example.description != null
+              ? `
+                <span>
+                  <div class="example-description">${example.description}</div>
+                  <div class="example-title">${example.title}</div>
+                </span>
```

Comment:
I would suggest removing one level in the tree and switching the `div` to `span` to simplify the the code.

```suggestion
                <span class="example-description">${example.description}</span>
                <span class="example-title">${example.title}</span>
```